examples/ES/mujoco_agent.py

```diff
@@ -37,7 +37,7 @@ def predict(self, obs):
         obs = np.expand_dims(obs, axis=0)
         obs = paddle.to_tensor(obs, dtype='float32')
         predict_actions = self.alg.predict(obs)
-        return predict_actions
+        return predict_actions.numpy()
```

Comment:
Do we need detach() here?

Reply:
As for paddle, it can runs well without adding the detach(), but add detach() seems more reasonably and same to the torch. Fixed it.